### PR TITLE
Added tests for the classical damage calculator

### PR DESCRIPTION
**Pull request by micheles**
The tests 6b, 7a, 7b, 7c from https://docs.google.com/a/globalquakemodel.org/document/d/1MD09_DVj6nqp2v_emeOi5lxZaytwc9mc6eOL52i7lS0/edit#heading=h.9jr0k7sqtgil.

This is the companion of https://github.com/gem/oq-risklib/pull/120.

### CODE REVIEW

File: qa_tests/risk/classical_damage/test.py [hunk 1]
```diff
@@ -29,20 +31,25 @@ class ClassicalDamageCase1aTestCase(risk.FixtureBasedQATestCase):
...
+            risk_calculation=job).order_by('lsi'))
+        data = block_splitter(
+            models.DamageData.objects.filter(
+                dmg_state__risk_calculation=job).order_by(
+                'exposure_data', 'dmg_state'),
+            len(damage_states))
+        # this is a test with 5 damage states
         # no_damage, slight, moderate, extreme, complete
-        return [row.fraction for row in data]
+        # print [[round(col.fraction, 8) for col in row] for row in data]
```
Comment (ptormene) [inline qa_tests/risk/classical_damage/test.py]: Probably we want to remove this commented print

File: qa_tests/risk/classical_damage/test.py [hunk 2]
```diff
@@ -114,4 +121,97 @@ class ClassicalDamageCase5aTestCase(ClassicalDamageCase1aTestCase):
...
+
+    def actual_data(self, job):
+        damage_states = list(models.DmgState.objects.filter(
+            risk_calculation=job).order_by('lsi'))
+        outs = models.Output.objects.filter(oq_job=job).order_by('id')
+        rows = []
+        for out in outs:
+            data = [[
+                col.fraction for col in row] for row in block_splitter(
+                models.DamageData.objects.filter(
```
Comment (ptormene) [inline qa_tests/risk/classical_damage/test.py]: Is this indentation ok?

Comment (micheles) [inline qa_tests/risk/classical_damage/test.py]: It is the only indentation I could find that makes flake8 happy.

**Comment by ptormene** [comment]
LGTM

**Pull request merged**